projects/saturn/src/test/java/io/fairspace/saturn/services/views/ViewServiceTest.java

```diff
@@ -107,11 +107,5 @@ public void testFetchViewConfig() {
                 .filter(facet -> facet.getType() == ViewsConfig.ColumnType.Date)
                 .collect(Collectors.toList());
         Assert.assertEquals(2, dateFacets.size());
-
-        var boolFacets = facets.stream()
```

Comment:
it looks like you added test-views.yaml introduced in the original PR, but you commented out the line, where it is used for tests. I guess this is why the tests fail for you,